packages/web/pages/index.js

```diff
@@ -20,16 +20,13 @@ const Home = ({ emailConfirmation }) => {
 	return (
 		<>
 			<Hero />
-			<TechnologiesSection
-				header={t('common:featuredSolutions')}
-				technologies={fullTechnologies}
-				bgColor={colors.gray98}
-			/>
-			<TechnologiesSection
-				header={t('common:recentSolutions')}
-				technologies={technologies}
-				bgColor={colors.whiteSmoke}
-			/>
+			{!!technologies?.length && (
+				<TechnologiesSection
+					header={t('common:recentSolutions')}
+					technologies={technologies}
+					bgColor={colors.whiteSmoke}
```

Comment:
Se as "Soluções em Destaque" forem em uma nova issue, gente pode ver nela. Mas se for para incluir nessa, gente pode só colocar cada seção com uma cor, para manter aquele diferença no background da seção :+1:

Reply:
Verdade, é bom lembrar disso para a próxima issue!